pyvista/demos/demos.py

```diff
@@ -212,20 +212,20 @@ def plot_wave(fps=30, frequency=1, wavetime=3, interactive=False, notebook=None)
 
     Parameters
     ----------
-    fps : int, optional
-        Maximum frames per second to display.  Defaults to 30.
+    fps : int, default: 30
+        Maximum frames per second to display.
 
-    frequency : float, optional
-        Wave cycles per second.  Defaults to 1 Hz.
+    frequency : float, default: 1.0
+        Wave cycles per second (Hz).
 
-    wavetime : float, optional
-        The desired total display time in seconds.  Defaults to 3 seconds.
+    wavetime : float, default: 3.0
+        The desired total display time in seconds.
 
-    interactive : bool, optional
+    interactive : bool, default: False
         Allows the user to set the camera position before the start of the
-        wave movement.  Default ``False``.
+        wave movement.
 
-    notebook : bool, optional
+    notebook : bool, default: None
```

Comment:
This should stay optional if default is correct here and it is unclear what happens in that case to me.  Based on the docstring alone and the docstring of Plotter I'm unsure whether we can just just default of `False`, which would be more clear.

This comment applies to 3 more changes too.

Reply:
```suggestion
    notebook : bool, optional
```